docs/proofs/example1.rst

```diff
@@ -0,0 +1,280 @@
+Example 1
+=========
+
+.. list-table:: Nat identity
+
+   * - As an example of elaborator reflection here we step through this identity implementation:
```

Comment:
I'm finding it quite hard to replace 'we' because 'I' is not appropriate due to the anonymous nature of the documentation so I am rewording sentences in a more impersonal way.